src/resource/file.ts

```diff
@@ -191,7 +213,24 @@ export async function saveFileInContainer<FileExt extends File | Buffer>(
   return Object.assign(blobClone, resourceInfo);
 }
 
+/**
+ * ```{note} The Web Access Control specification is not yet finalised. As such, this
+ * type is still experimental and subject to change, even in a non-major release.
```

Comment:
I don't think this has anything to do with WAC? :) 

```suggestion
 * ```{note} This type is still experimental and subject to change, even in a
 * non-major release.
```